packages/mobile/src/home/NotificationBox.tsx

```diff
@@ -250,7 +250,7 @@ export class NotificationBox extends React.Component<Props, State> {
                 notificationType: NotificationBannerTypes.remote_notification,
                 selectedAction: NotificationBannerCTATypes.remote_notification_cta,
               })
-              openUrl(notification.ctaUri)
+              openUrl(notification.ctaUri, true)
```

Comment:
I think the 2nd argument is for `openExternal`, and here you meant the safe origin one.
I propose we switch this to an options object to avoid incorrect usages.

Reply:
Yeah, I had fixed this here and in the test but forgot to push de commit 🤦  Thanks!